folly/portability/Windows.h

```diff
@@ -37,6 +37,7 @@
 #include <direct.h> // nolint
 #endif
 
+#define NOMINMAX
```

Comment:
I'm using folly via vcpkg w/ msbuild hence it's not picking up this, for me it's natural to include this in the header and not in the cmake file because not everyone is supposed to use cmake (think autoconf too).